masakari-controller/controller/masakari_util.py

```diff
@@ -53,6 +51,26 @@
 import db.api as dbapi
 from db.models import NotificationList, VmList, ReserveList
 
+from oslo_log import log as logging
+
+LOG = logging.getLogger(__name__)
+
+class LogProcessBeginAndEnd(object):
+    def output_log(self, func):
+        def _output_log(*args, **kwargs):
+            start_msg = ("BEGIN %s: parameters is %s, %s") % (func, args, kwargs)
```

Comment:
Use func.__name__ instead of func.  Specifying 'func' shows method name and memory address. it's no information.